### DISCUSSION TITLE
Difference between this image and the orange pi official

### DISCUSSION THREAD

**Discussion by vossenv** [Joshua-Riek/ubuntu-rockchip#322] [Q&A] [answered]
Hey all - new to OPI but not SBC in general.   It seems there's also an "official" ubuntu image for the pi 5.  Can someone give me a summary of why I might want to use this image vs that one? Is this one further optimized / fixed in some way?  I'm just a little confused at the relationship since this is also listed as a 3rd party on their site

**Comment by Joshua-Riek** [comment, marked as answer]
I started this project because I did not like the "official" Ubuntu image from Orange Pi, so I decided to build my own from scratch. I have listed below some of the differences that stand out to the end user. However, it does not include all differences.  

### The "official" Ubuntu image:
- The image is an outdated fork of Armbian; everyone's definition of "ubuntu" differs
- Users can easily break the hardware acceleration system by removing package holds
- Outdated GPU drivers and only partially uses panfork mesa
- The BSP kernel used has no commit history and is now considered old
- User bug reports, and issues are generally never addressed
- There is no way to update the image 
- Orange Pi has no incentive to maintain or work on the image unless it's for adding a new SBC

### My Ubuntu image:
- Package management via apt using the official Ubuntu repositories
- Receive kernel, firmware, and bootloader updates through apt
- Desktop first-run wizard for user setup and configuration
- The CPU and GPU are optimized right out of the box
- 3D video hardware acceleration support via panfork thanks to @amazingfate's ppa
- Latest available BSP kernel in collaboration with @armbian, we have a lot of bug fixes and improvements
- Updates, improvements, and bug fixes are being worked on constantly

Overall I have received a lot of positive feedback from users about my image. Generally, performance is better, and users will face a more traditional Ubuntu experience. The goal is to have a Ubuntu image similar to what you might see on the popular Raspberry Pi 4.

**Reply by vossenv** [reply]
Thanks for taking the time to reply! This all sounds really great - and on a side note, I was able to get your image working more easily for me.  Thanks for what you're doing here!

**Reply by rgborn** [reply]
You are doing good work here.  Thanks!

**Reply by douglasdock** [reply]
thanks for the work

**Reply by hqnicolas** [reply]
Thanks for your work! this is an oasis for rockchip devices
